vscode/src/status.ts

```diff
@@ -178,6 +178,31 @@ export class FormatterStatus extends StatusItem {
   }
 }
 
+export class AddonsStatus extends StatusItem {
+  constructor() {
+    super("addons");
+
+    this.item.name = "Ruby LSP Addons";
+    this.item.text = "Fetching addon information";
+  }
+
+  refresh(workspace: WorkspaceInterface): void {
+    if (workspace.lspClient) {
+      if (workspace.lspClient.addons === undefined) {
+        this.item.text =
+          "Addons: requires server to be v0.17.4 or higher to display this field";
+      } else if (workspace.lspClient.addons.length === 0) {
+        this.item.text = "Addons: none";
+      } else {
+        const addonNames = workspace.lspClient.addons.map((addon) =>
+          addon.errored ? `${addon.name} (errored)` : `${addon.name}`,
+        );
+        this.item.text = `Addons: ${addonNames.join(", ")}`;
```

Comment:
Instead of showing all names concatenated by commas, could we started with a quick pick that does nothing instead? The language status items will all expand to fit the widest entry and two addon names is already quite long `Ruby LSP Rails, Ruby LSP RSpec`.

It will also allow you to show the information about whether it errored or not in a [quick pick detail](https://code.visualstudio.com/api/references/vscode-api#QuickPickOptions), which will look nicer.

Reply:
That's okay.